daprdocs/content/en/python-sdk-docs/python-client.md

```diff
@@ -8,16 +8,18 @@ description: How to get up and running with the Dapr Python SDK
 
 The Dapr client package allows you to interact with other Dapr applications from a Python application.
 
-## Pre-requisites
+{{% alert title="Note" color="primary" %}}
+ If you haven't already, [try out one of the quickstarts]({{< ref quickstarts >}}) for a quick walk-through on how to use the Dapr Python SDK with an API building block.
 
-- [Dapr CLI]({{< ref install-dapr-cli.md >}}) installed
-- Initialized [Dapr environment]({{< ref install-dapr-selfhost.md >}})
-- [Python 3.7+](https://www.python.org/downloads/) installed
-- [Dapr Python module]({{< ref "python#install-the0dapr-module" >}}) installed
+{{% /alert %}}
+
+## Prerequisites
+
+[Install the Dapr Python module]({{< ref "python-sdk-docs/_index.md" >}}) before getting started.
 
 ## Import the client package
 
-The dapr package contains the `DaprClient` which will be used to create and use a client.
+The Dapr package contains the `DaprClient`, which is used to create and use a client.
```

Comment:
Maybe we should use lowercase dapr to match the import. By using the code formatting we can indicate this is intentional.

```suggestion
The `dapr` package contains the `DaprClient`, which is used to create and use a client.
```